source/_docs/php-errors.md

```diff
@@ -15,90 +15,90 @@ For more in-depth information, see [Error Handling and Logging](https://secure.p
 Each of the PHP errors are handled differently depending on the site environment. On Dev, they are shown directly to the user in the browser. On Test and Live, PHP errors are not displayed to users, but they'll still be logged. Notices and warnings are logged in the database logs if `db\_log` is enabled for Drupal. The PHP constants `WP_DEBUG` and `WP_DEBUG_LOG` can be enabled for WordPress to save errors to wp-content/debug.log. PHP errors are also logged on the application server at `logs/php-error.log`.
 
 Here's a breakdown of what errors are shown and where:
-<table>

```

Comment:
Note: The only changes here are Y => ✓ and N => " ". The entire block is red because I removed CRLFs.

Reply:
Any reason not to go ahead and convert this to markdown while we're here? (defer to you on if converting all tables to markdown is in scope of this PR but seems like we could do it quick since this file is in the diff now.)

Reply:
Any reason not to go ahead and convert this to markdown while we're here? (defer to you on if converting all tables to markdown is in scope of this PR but seems like we could do it quick since this file is in the diff now.)